packages/yoastseo/spec/scoring/assessments/inclusiveLanguage/configuration/otherAssessmentsSpec.js

```diff
@@ -0,0 +1,85 @@
+import Paper from "../../../../../src/values/Paper";
+import EnglishResearcher from "../../../../../src/languageProcessing/languages/en/Researcher";
+import InclusiveLanguageAssessment from "../../../../../src/scoring/assessments/inclusiveLanguage/InclusiveLanguageAssessment";
+import assessments from "../../../../../src/scoring/assessments/inclusiveLanguage/configuration/otherAssessments";
+
+describe( "Age assessments", function() {
+	it( "should target non-inclusive phrases",
+		function() {
+			const mockPaper = new Paper( "This ad is aimed at homosexuals" );
+			const mockResearcher = new EnglishResearcher( mockPaper );
+			const assessor = new InclusiveLanguageAssessment( assessments.find( obj => obj.identifier === "homosexuals" ) );
+
+			const isApplicable = assessor.isApplicable( mockPaper, mockResearcher );
+			const assessmentResult = assessor.getResult();
+
+			expect( isApplicable ).toBeTruthy();
+			expect( assessmentResult.getScore() ).toEqual( 6 );
+			expect( assessmentResult.getText() ).toEqual(
+				"Be careful when using <i>homosexuals</i> as it may overgeneralize or be harmful. " +
+				"Instead, be specific about the group you are referring to (e.g. <i>gay men, queer people, lesbians</i>). " +
+				"<a href='https://yoa.st/4ls' target='_blank'>Learn more.</a>" );
```

Comment:
```suggestion
			expect( assessmentResult.getText() ).toEqual(
				"Be careful when using <i>homosexuals</i> as it may overgeneralize or be harmful. " +
				"Instead, be specific about the group you are referring to (e.g. <i>gay men, queer people, lesbians</i>). " +
				"<a href='https://yoa.st/inclusive-language-other' target='_blank'>Learn more.</a>" );
```